front-end/src/components/Edit/AddEducation.js

```diff
@@ -59,13 +74,26 @@ class AddEducation extends Component{
               <div style = {this.props.spacer}>
                 Start Date<input style={this.props.inputStyle} type="date" key="Start Date" value={this.state.education.start_date} onChange={(e) => this.updateStartDate(e.target.value)}/>
               </div>
-              <div style = {this.props.spacer}>
-                End Date<input style={this.props.inputStyle} type="date" key="End Date" value={this.state.education.end_date} onChange={(e) => this.updateEndDate(e.target.value)}/>
-              </div>
+                {
+                  this.state.checked === true
+                  ?
+                  <div style = {this.props.spacer}>
+                    <input type="checkbox" checked={this.state.checked} onChange={()=>{this.updateCheckbox()}}/>I currently attend here
```

Comment:
I like the checkbox idea, but we're repeating the exact same snip of code twice. Maybe defining a pure function to return this div will be a bit cleaner? We still need to clean up a bunch of code to avoid repetition, but we can start with this one. What do you think?

Reply:
Good point. I'll refactor that to be a stateless component and update the PR. Probably later this evening though.